game_event/views.py

```diff
@@ -1 +1,38 @@
-# from django.shortcuts import render
+from .models import GameEvent
+from django.shortcuts import render, redirect
+from court.models import Court
+from player_rating.models import Rating
+from player.models import BallGame
+from django.utils import timezone
+from django.core.exceptions import ValidationError
+from django.contrib import messages
+
+
+def display_game_event_form(request):
```

Comment:
missing tests.

Reply:
Also, by having the same function used for both http methods, you're making this code harder to test. i.e. harder to maintain.

Reply:
took me some modifications, but i managed to split it to seperate views

Reply:
added test for display the form, and for post process